dokang/utils.py

```diff
@@ -52,8 +58,7 @@ def load_doc_sets(settings):
     if not os.path.exists(upload_dir):
         os.makedirs(upload_dir)
 
-    sets = settings['dokang.doc_sets'] = {}
-    for uploaded in os.listdir(upload_dir):
-        sets[uploaded] = doc_set(settings, uploaded)
-
-    return settings
+    return {
+        uploaded: doc_set(settings, uploaded)
+        for uploaded in os.listdir(upload_dir)
+    }
```

Comment:
Since the function now returns something without changing the state (of the settings), perhaps we should rename it as `get_doc_sets()` ("get" instead of "load")?